packages/medusa-payment-stripe/src/services/stripe-ideal.js

```diff
@@ -1,15 +1,11 @@
-import _ from "lodash"
 import Stripe from "stripe"
-import { PaymentService } from "medusa-interfaces"
+import { AbstractPaymentService, PaymentSessionStatus } from "@medusajs/medusa"
 
-class IdealProviderService extends PaymentService {
-  static identifier = "stripe-ideal"
+class StripeProviderService extends AbstractPaymentService {
```

Comment:
question: Why does this file contain this many changes? It has also been renamed. I believe this should almost be identical to what it was before, but I might be mistaken.

Reply:
certainly a wrongly managed conflict 👍

Reply:
All good 💪